packages/components/psammead-sitewide-links/src/index.test.jsx

```diff
@@ -23,6 +36,16 @@ describe(`SitewideLinks`, () => {
     />,
   );
 
+  shouldMatchSnapshot(
+    'should render correctly with CCPA link',
+    <SitewideLinks
+      links={ccpaLinks}
+      copyrightText={<span>Text here.</span>}
+      externalLink={link}
+      service="news"
+    />,
+  );
+
```

Comment:
I'm not sure it's necessary to have an additional test for this. Can I suggest we include the new link in the default `links` array, so it's covered by the original test?